public/apps/configuration/panels/role-mapping/types.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import { EuiComboBoxOptionOption } from '@elastic/eui';
+
+export type ComboBoxOptions = EuiComboBoxOptionOption[];
```

Comment:
I lifted the same type to upper folder level in recent changes:
https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/blob/67661c972465e90dc1295da9302f72c4c035046a/public/apps/configuration/types.ts#L16-L18

Let's share the same one.

Reply:
corrected.